dev-requirements.txt

```diff
@@ -1,9 +1,3 @@
 mock==1.0.1
 pytest-cov==2.5.1
 coverage==3.7.1
-boto3>=1.7.28
-#google-cloud==0.33.1
-google-cloud-storage==1.6.0
--e git+https://git@github.com/uc-cdis/cdislogging.git@0.0.2#egg=cdislogging
--e git+https://git@github.com/uc-cdis/indexclient.git@1.5.9#egg=indexclient
```

Comment:
why did these get removed? Also, do you know the indexclient a zip file in the repo?

Reply:
oh sorry, looks like these just got moved to the other req file

Reply:
i removed them because they were already in `requirements.txt`

indexclient has to be a zip file because it's not available outside of github, https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/
I think we should just add it to pypi but Giang mentioned we would need to test it which is expensive, so leaving it as a zip file for now